TransitionsApp/SecondViewController.swift

```diff
@@ -9,34 +9,53 @@ import Foundation
 import UIKit
 
 final class SecondViewController: UIViewController {
-    @IBOutlet private weak var colorButton: UIButton!
+    @IBOutlet private weak var redButton: UIButton!
     @IBOutlet private weak var closeButton: UIButton!
-    
-    var nameButton = "カラー"
+    @IBOutlet weak var greenButton: UIButton!
+    @IBOutlet weak var blueButton: UIButton!
+    var redColorButton = "赤"
     var buttonName = "戻る"
+    var blueColorButton = "青"
+    var greenColorButton = "緑"
     
     override func viewDidLoad() {
         super.viewDidLoad()
         secondchangeNameButton()
         thirdchangeNameButton()
-        
+        forcecangeNameButton()
+        fifthchangeNameButton()
+        let selectedColor: colour = .blue
+        self.view.backgroundColor = selectedColor.colorSetting
     }
     func secondchangeNameButton(){
-        colorButton.setTitle(nameButton, for: .normal)
+        redButton.setTitle(redColorButton, for: .normal)
     }
     func thirdchangeNameButton(){
         closeButton.setTitle(buttonName, for: .normal)
     }
-
-    @IBAction func buttonTapped(_ sender: Any) {
-        view.backgroundColor = UIColor.yellow
+    func forcecangeNameButton(){
+        blueButton.setTitle(blueColorButton, for: .normal)
+    }
+    func fifthchangeNameButton(){
+        greenButton.setTitle(greenColorButton, for: .normal)
     }
     
     @IBAction func secondButtonTapped(_ sender: Any) {
         self.dismiss(animated: true, completion: nil)
     }
 }
 
-
-
+enum colour{
+    case blue,red,green
+    var colorSetting: UIColor{
+        switch self {
+        case.blue:
+            return.blue
+        case .red:
+            return.red
+        case .green:
+            return.green
+        }
+    }
+}
```

Comment:
この書き方もできなくはないけど、
```suggestion
enum Color {
    case red
    case blue
    case green
    
    var colorSetting: UIColor {
        switch self {
        case .red:
            return .red
        case .blue:
            return .blue
        case .green:
            return .green
        }
    }
}
```

のようにcaseは1つずつ書くのと、enumは最初は大文字で書く！

あと、これは説明してなかったんだけど、enumの処理は10行目に書く方がいいかな！